src/Schema/Directives/WhereDirective.php

```diff
@@ -34,12 +36,13 @@ public static function definition(): string
 
     /**
      * Add any "WHERE" clause to the builder.
-     *
-     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder  $builder
-     * @return \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder
      */
     public function handleBuilder($builder, $value): object
     {
+        if ($builder instanceof ScoutBuilder) {
+            throw new Exception("Using {$this->name()} on queries that use a Scout search is not supported.");
```

Comment:
Yeah, we definitely have to refactor this to a custom exception, so this string is not duplicated all the time 👍